railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt

```diff
@@ -8,6 +8,11 @@ max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
 min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
 threads min_threads_count, max_threads_count
 
+# Specifies the `worker_timeout` threshold that Puma will use to wait before
+# terminating a worker in development environments.
+#
+worker_timeout 3600 if ENV.fetch("RAILS_ENV") == "development"
```

Comment:
`RAILS_ENV` can be unset so `ENV.fetch("RAILS_ENV")` will fail. I think we should default to "development" inside a block.

Reply:
Will this work? Or is the block format better?
```suggestion
worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
```

Reply:
This also work. Feel free to apply and make sure all commits are squashed in one.